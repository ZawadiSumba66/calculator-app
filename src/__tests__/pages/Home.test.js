import renderer from 'react-test-renderer'
import Home from '../../components/Home'
import React from 'react'


it('matches the Home snapshot',()=>{
   const tree= renderer.create(<Home/>).toJSON()
   expect(tree).toMatchSnapshot()
})
