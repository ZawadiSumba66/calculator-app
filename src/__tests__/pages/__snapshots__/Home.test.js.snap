// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the Home snapshot 1`] = `
<div
  className="center"
>
  <h1>
    Welcome to our page!
  </h1>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
  </p>
  <p>
    Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. IpsuLorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
  </p>
  <p>
    Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
  </p>
</div>
`;
